pyangbind/lib/yangtypes.py

```diff
@@ -895,7 +900,11 @@ def YANGDynClass(*args, **kwargs):
   if not base_type:
     raise TypeError("must have a base type")
 
-  if isinstance(base_type, list):
+  if base_type == bitarray:
```

Comment:
Can you add a comment explaining why this is required please? It's not immediately obvious.